jsonenums.go

```diff
@@ -75,6 +75,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"./utils"
```

Comment:
Probably easier to discuss this f2f but what happens if you change this line to "utils"? That's how the package should be imported.